.github/workflows/plugin_install.yml

```diff
@@ -3,7 +3,7 @@ name: Plugin Install
 on: [push, pull_request, workflow_dispatch]
 
 env:
-  OPENSEARCH_VERSION: 2.4.1
+  OPENSEARCH_VERSION: 2.4.2
```

Comment:
You could automate this by setting this as an ID and then using something along the lines of:  

```
awk '*-SNAPSHOT' ./build.gradle | awk -F '-' $OPENSEARCH_VERSION=$1
```

This should parse build.gradle for the string containing the term "-SNAPSHOT" and then pipe that line over where it is split with the deliminator  '-' and set the variable to be the first of the substrings. 

This may not be perfect syntax but should be close.